.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Report a bug
+title: ''
+labels: 'T: bug'
+assignees: ''
+---
+
+**Description**
+Add a clear and concise description of what the bug is. Example: Capturing metrics manually does not work properly.
```

Comment:
Perhaps instructions should be markdown comments, so you don't have to delete them?

Reply:
Also had this in mind. Will convert them to comments 😄